02_Util/src/certificate/CertificateUtil.ts

```diff
@@ -0,0 +1,245 @@
+// Copyright Contributors to the CitrineOS Project
```

Comment:
Centralize certificate related functions into this util file. Just copy from other scripts and paste there. No new changes.